src/Aspire.Dashboard/Model/ResourceSourceViewModel.cs

```diff
@@ -3,45 +3,74 @@
 
 namespace Aspire.Dashboard.Model;
 
-public class ResourceSourceViewModel(string value, string? contentAfterValue, string valueToVisualize, string tooltip)
+public class ResourceSourceViewModel(string value, List<LaunchArgument>? contentAfterValue, string valueToVisualize, string tooltip)
 {
     public string Value { get; } = value;
-    public string? ContentAfterValue { get; } = contentAfterValue;
+    public List<LaunchArgument>? ContentAfterValue { get; } = contentAfterValue;
     public string ValueToVisualize { get; } = valueToVisualize;
     public string Tooltip { get; } = tooltip;
 
     internal static ResourceSourceViewModel? GetSourceViewModel(ResourceViewModel resource)
     {
-        var executablePath = resource.TryGetExecutablePath(out var path) ? path : null;
+        (List<LaunchArgument>? Arguments, string? ArgumentsString)? commandLineInfo;
```

Comment:
Should this be non-nullable? I can't see where it's ever assigned to null.

```suggestion
        (List<LaunchArgument>? Arguments, string? ArgumentsString) commandLineInfo;
```

Some of the code below does actually check for null, but I don't think there's a problem.